core/designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/DynamicAsyncImage.kt

```diff
@@ -34,11 +44,27 @@ fun DynamicAsyncImage(
     placeholder: Painter? = null,
 ) {
     val iconTint = LocalTintTheme.current.iconTint
-    AsyncImage(
-        placeholder = placeholder,
+    SubcomposeAsyncImage(
+        error = {
+            Image(
+                painter = placeholder ?: painterResource(R.drawable.ic_placeholder_default),
```

Comment:
Lift `placeholder ?: painterResource(R.drawable.ic_placeholder_default)` as a default value in the method signature

Reply:
Done